posts/2024-12-10_road-to-mainnet/README.md

```diff
@@ -0,0 +1,343 @@
+---
+publication_date: 2024-12-10T00:00:00Z
+slug: road-to-mainnet
+tags: [gnoland]
+authors: [gnoland-team]
+---
+
+# gno.land: The Road to Mainnet & Beyond
+
+**Gno the Difference**
```

Comment:
```suggestion
## Gno the Difference
```

Reply:
No, I think this is fine the way it is; it shouldn't pop up in the ToC - its a subtitle

Reply:
It's a sub-heading isn't it?